### PR TITLE
Chore: use bulk method instead of iteration for performance

### PR DESCRIPTION
**Pull request by dbmalkovsky**
What started out as just using a bulk method, `addAll()` turned into removing a unused variable and fixing the name of another class variable so it didn't start with an uppercase `J`.

### CODE REVIEW

File: src/main/java/org/isf/medstockmovtype/gui/MedicalsrMovPatList.java [hunk 1]
```diff
@@ -47,10 +47,9 @@
 	private Patient myPatient;
 	private ArrayList<MovementWard> drugsData;
 	private JDialog dialogDrug;
-	private JTable JtableData;
+	private JTable jTableData;
 	private OhTableDrugsModel<MovementWard> modelMedWard;
 	private OhDefaultCellRenderer cellRenderer = new OhDefaultCellRenderer();
-	private MovWardBrowserManager movManager = Context.getApplicationContext().getBean(MovWardBrowserManager.class);
```
Comment (mwithi) [inline src/main/java/org/isf/medstockmovtype/gui/MedicalsrMovPatList.java:53]: I find it is better to have all managers loaded at the begin of the class, wdyt?

Comment (dbmalkovsky) [reply src/main/java/org/isf/medstockmovtype/gui/MedicalsrMovPatList.java:53]: Sure if it is being used but it appears not in this case so it is just a performance hit, isn't it?

Comment (mwithi) [reply src/main/java/org/isf/medstockmovtype/gui/MedicalsrMovPatList.java:53]: it is, at line 71, no?

Comment (mwithi) [reply src/main/java/org/isf/medstockmovtype/gui/MedicalsrMovPatList.java:53]: I mean, it was enough to remove line 68

Comment (dbmalkovsky) [reply src/main/java/org/isf/medstockmovtype/gui/MedicalsrMovPatList.java:53]: Understand now; fixed.

Comment (mwithi) [reply src/main/java/org/isf/medstockmovtype/gui/MedicalsrMovPatList.java:53]: alright! thanks!

**Review by mwithi — APPROVED** [review]
LGTM. Thanks a lot!

**Pull request merged**